website/views/shared/tasks/lists.jade

```diff
@@ -24,9 +24,14 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
 
           // Add New
           form.task-add(name='new{{list.type}}form', ng-hide='obj._locked', ng-submit='addTask(obj[list.type+"s"],list)')
-            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', required)
+            textarea(rows='6', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', required)
+            input(type='text', ng-model='list.newTask', placeholder='{{list.placeHolder}}', ng-if='!list.bulk', required)
             button(type='submit', ng-disabled='new{{list.type}}form.$invalid')
               span.glyphicon.glyphicon-plus
+            span.help-block
```

Comment:
That looks good. I will make those changes along with pulling in your CSS this afternoon. I am home sick from work today, so I should be able to update the PR in the next couple hours. I am not at my computer presently, so I can't do it this minute.